src/main/kotlin/no/skatteetaten/aurora/gradle/plugins/extensions/AuroraExtension.kt

```diff
@@ -331,3 +331,8 @@ fun Project.getFeaturesExtension(): Features {
 
     return (extension as ExtensionAware).extensions.getByType(Features::class.java)
 }
+
+fun Project.isKotlinStdlibEnabled(): Boolean {
+    val stdlib = project.properties["kotlin.stdlib.default.dependency"] as String?
+    return stdlib == null || stdlib == "enabled"
```

Comment:
Så i kotlin kan en sette "definisjonsmengden" for det en funksjon skal returnere? -kult :)